cmd/tetragon/main.go

```diff
@@ -190,6 +190,11 @@ func tetragonExecute() error {
 	option.Config.BpfDir = observerDir
 	option.Config.MapDir = observerDir
 
+	// Check if option to remove old BPF and maps is enabled.
+	if option.Config.ReleasePinned {
+		os.RemoveAll(observerDir)
```

Comment:
Do we need an option? Is there a reason to not just always do this?

Reply:
At some point enforced maps and progs need to stay loaded across pod restart.